### DISCUSSION TITLE
What about the weather?

### DISCUSSION THREAD

**Discussion by regnans** [ess-dive-community/essdive-uas#5] [General]
How should weather conditions during flights be recorded? What are the most important aspects to capture? Does it depend on the sensor suite deployed?

**Comment by freemansw1** [comment]
Do you mean _in situ_ weather conditions? I would think this would heavily depend on the sensor suite available and the postprocessing available. For example, wind measurements from multirotor UAS are really in their infancy (although there have been some really great recent efforts on this), so I would be hesitant to introduce a standard for storing them without the measurements themselves (and accuracy, uncertainty, etc.) are better quantified.

**Reply by serbinsh** [reply]
@freemansw1 In this case I think we referring to "in-situ", as in the general conditions in the flight area, perhaps collected from a nearby weather station and/or using basic visual documentation, including but not limited to, making notes about conditions like sunny, partly cloudy, etc etc.  Perhaps noting temperature and windspeeds using a handheld kestrel or similar.  

At a minimum collecting some info regarding what the conditions were during a data collection. For some data, like spectroscopy or thermal IR, it matters more to know air temperature, radiation, etc

Of course it could get more sophisticated like collecting met data on the platform but for now we are thinking about how to capture general condition info in the common metadata reporting format

**Reply by freemansw1** [reply]
Ah, that makes sense! Then yes, I generally agree that it's important to capture, but for some applications, this can vary substantially over the course of the flight. I think a basic weather conditions at takeoff would be a good way to balance that though, if it's acknowledged that the weather is only at the single point of takeoff.

**Reply by MatthiasSiewert** [reply]
Regarding weather, we record the following parameters:

Wind speed (m/s) (together with battery data (Number/%discharge, flight time)

Lightness (bright/dim/dark)
Light in Lux (lx- currently only from an app on the phone))
Sun/Cloud (stable/changing)
Light type (direct/diffuse/changing)

Cloud cover (%)
Cloud type (stratos, cirrus, cumulus)

Wetness on ground(dry/moist/wet)
Days since last rain event.

**Reply by regnans** [reply]
Matthias, thanks for sharing. That's a useful list. Depending on application some people with have records from local weather stations or light meters, but these categorical descriptions will be a good approach when quantitative measures are not available (or needed).
